UI/html/adminrequest.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Admin requests</title>
+
+  <link href='https://fonts.googleapis.com/css?family=Old Standard TT' rel='stylesheet'>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+  <link rel="stylesheet" href="./../styles/adminrequest.css" type="text/css">
+</head>
+<body>
+  <div class="header">
+    <h1>Maintenance Tracker <i class="fas fa-cogs"></i></h1>
+  </div>
+  <div id="filter">
+    <div><h4>Filter: </h4></div>
+    <div id="sort">
+      <select name="req" id="req">
+        <option value="repair">Repairs</option>
+        <option value="maintenance">Maintenance</option>
+      </select>
+    </div>
+  </div>
+  
+  <div class="container">
+    <div class="requests">
+      <p>Request by: Andrew Davids</p>
+      <p>Position: Clerk</p>
+      <p>Type: Maintenance</p>
+      <a href="#">more details</a>
```

Comment:
The `more details` link is repeated too much. It would be better to make the entire card clickable to show the details.